testsuite/integration-tests/pom.xml

```diff
@@ -88,13 +88,16 @@
                                             <name>domain</name>
                                         </excluded-packages>
                                     </feature-pack>
-                                    <feature-pack>
-                                        <groupId>${resteasy.test.feature.pack.groupId}</groupId>
-                                        <artifactId>${resteasy.test.feature.pack.artifactId}</artifactId>
-                                        <version>${resteasy.test.feature.pack.version}</version>
-                                        <inherit-configs>false</inherit-configs>
-                                    </feature-pack>
                                 </feature-packs>
+                                <channels>
```

Comment:
If you're able to use newer WildFly (>= 32.0.0.Final) it might be worth adding another channel with the WildFly manifest (`org.wildfly.channels:wildfly-ee`). Server provisioning should be much faster this way.